client/lib/application/src/version-form/version-form.js

```diff
@@ -96,7 +96,7 @@ class VersionForm extends BaseView {
             .props
             .notificationActions
             .addNotification(
-                `Could not ${verb} version ${version_id} for ${this.data.application.name}. ${err.message}`,
+                `Could not ${verb} version ${version_id} of ${this.data.application.name}. ${err.message}`,
```

Comment:
Yup, true.

----- Ursprüngliche Nachricht -----
Von: "Emanuel Lauria" <notifications@github.com>
Gesendet: ‎19.‎05.‎2015 18:00
An: "zalando-stups/yourturn" <yourturn@noreply.github.com>
Cc: "Nikolaus Piccolotto" <prayerslayer@gmail.com>
Betreff: Re: [yourturn] Update approvals (#103)

In client/lib/application/src/version-form/version-form.js:
> @@ -96,7 +96,7 @@ class VersionForm extends BaseView {
>              .props
>              .notificationActions
>              .addNotification(
> -                `Could not ${verb} version ${version_id} for ${this.data.application.name}. ${err.message}`,
> +                `Could not ${verb} version ${version_id} of ${this.data.application.name}. ${err.message}`,
should be camelCase everywhere
—
Reply to this email directly or view it on GitHub.